packages/phoenix-evals/src/phoenix/evals/llm/adapters/google/adapter.py

```diff
@@ -56,7 +56,7 @@ def _check_if_async_client(self) -> bool:
             return False
         return True
 
-    def generate_text(self, prompt: Union[str, MultimodalPrompt], **kwargs: Any) -> str:
+    def generate_text(self, prompt: Union[str, List[Dict[str, Any]]], **kwargs: Any) -> str:
```

Comment:
```suggestion
    def generate_text(self, prompt: PromptLike, **kwargs: Any) -> str:
```

Might consider type aliasing just to make it a bit more readable.